homeassistant/util/unit_system.py

```diff
@@ -250,6 +250,8 @@ def _deprecated_unit_system(value: str) -> str:
     _CONF_UNIT_SYSTEM_METRIC,
     accumulated_precipitation=UnitOfPrecipitationDepth.MILLIMETERS,
     conversions={
+        # Convert non-metric atmospheric pressures
+        ("atmospheric_pressure", UnitOfPressure.INHG): UnitOfPressure.HPA,
```

Comment:
Should we convert any pressure which is not `hPa` to `hPa`? Same comment for USCS, but with conversion to `InHg`.

Reply:
done